src/System.Runtime.Intrinsics.Experimental/System.Runtime.Intrinsics.Experimental.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27419.1
+# Visual Studio 14
```

Comment:
Yeah, but typically folks shouldn't be modifying these SLNs, we auto-generate them.  If the auto-generation is missing something we should fix that.

Reply:
Right, but VS will mark them dirty, and want to save them, and then they need to be reverted locally. NBD but at some point we should change the autogenerator to 15.

Reply:
I agree, but it will never be a "good" value for that satisfies everyone.  We'll always have folks on different versions of VS and that would create unnecessary churn if they need to change the sln.  My point was that we should try to come up a system where folks never need to change the sln that way we don't have to worry about churn on the versions.

Reply:
As part of cleaning up VS2015 support we should update the generator. I added a note about that here https://github.com/dotnet/corefx/issues/24320#issuecomment-371559895